src/server/scripts/EasternKingdoms/BlackrockMountain/BlackwingLair/boss_chromaggus.cpp

```diff
@@ -327,6 +325,9 @@ class go_chromaggus_lever : public GameObjectScript
 
                 return true;
             }
+        
+        private:
+            InstanceScript* instance;
```

Comment:
@Keader you sucker, it's _instance; It's a private member. Fix that and this should be good to go.